.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: minimal
+branches:
+  only:
+  - develop
+  - master
+git:
+  depth: false
+services:
+- docker
+install:
+- eval $(ssh-agent -s)
```

Comment:
`eval "$(ssh-agent -s)"`
Github recommends this form.
https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent